packages/tradershub/src/providers/SignupWizardProvider/SignupWizardContext.tsx

```diff
@@ -49,6 +25,7 @@ const initialHelpers: Helpers = {
         /* noop */
```

Comment:
what's this noop for?

Reply:
![Uploading Screenshot 2024-01-31 at 4.45.26 PM.png…]()

Reply:
![Screenshot 2024-01-31 at 4 45 26 PM](https://github.com/binary-com/deriv-app/assets/104053934/e2ca1b8d-f0ad-4e25-8624-6bf78e35978e)

Reply:
This is to define the initial state for the context as an empty function due to our typescript rule
`Unexpected empty arrow function 'goToPrevStep'.eslint[@typescript-eslint/no-empty-function](https://typescript-eslint.io/rules/no-empty-function)`